### ISSUE TITLE
Conflit between use_fused_kernels and use_liger

### ISSUE THREAD

**Issue by zengwz** [verl-project/verl#1720]
If set actor_rollout_ref.model.use_fused_kernels=True and actor_rollout_ref.model.use_liger=True at the same time, the loss and reward have a big gap.

<img width="342" alt="Image" src="https://github.com/user-attachments/assets/f727449b-84c7-449b-9144-cc7878a18545" />

<img width="350" alt="Image" src="https://github.com/user-attachments/assets/c566bfbd-319d-4c85-a67f-0a93652003cb" />

<img width="343" alt="Image" src="https://github.com/user-attachments/assets/86662c68-ccb2-457d-bbb9-06e335e8630d" />

<img width="347" alt="Image" src="https://github.com/user-attachments/assets/3421e177-1461-4006-92b4-de80b7337077" />

blue: baseline
orange: use_fused_kernels=True
red: use_fused_kernels=True and use_liger=True

Config:
model: qwen_2_5_vl_7b
dataset: geo3k

By the way, I find that liger repo has the FusedLinearCrossEntropyLoss function to reduce peak memory by chunking data, similar to the fused kernels commit. Is there a elegant way to implement this fused kernel with use liger?
https://github.com/linkedin/Liger-Kernel/blob/ea3ac1b86a853e56d3fec75791bda061550a8e1c/src/liger_kernel/transformers/fused_linear_cross_entropy.py#L8

**Comment by ETOgaosion** [comment]
Thanks for reporting this. You are right, both liger and fused kernel are trying to fuse the last layer and cross entropy calculations, they should not work together, we will make liger as a fused option.

**Comment by vadimkantorov** [comment]
> Thanks for reporting this. You are right, both liger and fused kernel are trying to fuse the last layer and cross entropy calculations, they should not work together, we will make liger as a fused option.

Should `_apply_liger_kernel_to_instance(model=self.model)` be instead `_apply_liger_kernel_to_instance(model=self.model, fused_linear_cross_entropy=not use_fused_kernels)`?

So that if verl's `use_fused_kernels` is set to True, Liger does not interfere in replacing model's forward?

**Comment by vadimkantorov** [comment]
Also, should Verl do something like chunked GRPO? 

Also, if entropy_coef=0, should entropy calc use torch.no_grad to not save the intermediate activations?

And also, should all loss calculation be moved to a torch.compile-d function?

**Comment by vadimkantorov** [comment]
Basically wondering, for peak VRAM minimization, should we focus on optimizing LinearForPPO returning log_probs/entropy or on making chunked GRPO?

Also, for some reason Embedding backward / FSDP's post backward grad hook is allocating a lot (making a peak larger than the backward beginning one)
